src/test/java/ch/zhaw/psit4/helper/CompanyGenerator.java

```diff
@@ -10,16 +10,19 @@
     public CompanyGenerator() {
```

Comment:
If this class only holds static methods, make class final. Make constructor private.